assignments/arrays.js

```diff
@@ -75,30 +75,70 @@ let inventory = [
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
```

Comment:
You got the answer correctly here, but your implementation could be a little better. You could find a way to implement this more programtically

Reply:
I tried to get the ID of the 33rd car in the array but this was the only way I could find. If there's another way I'll appreciate if you can show me. Thank you!